src/ProjectTemplates/test/BaselineTest.cs

```diff
@@ -0,0 +1,137 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Templates.Test.Helpers;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Templates.Test
+{
+    public class BaselineTest
```

Comment:
This file is already present in the `master` branch. I'm not sure why it was removed from the `blazor-wasm` branch. Assuming the tests can pass, it's probably desirable to keep this file and the baselines file the same as in `master` so a future merge is easier.